projectList

```diff
@@ -0,0 +1,19 @@
+Name - URL - @ addedby
+Crisis.app - https://github.com/CrisisRelief/crisisapp-marketplace - @BrianHHill
+osfm8 - https://www.laughingmind.com/osfm8 -  @BrianHHill
+ausbills - https://github.com/OpenGovAus/Aus-Bills - @KipCrossing
+legistream - https://github.com/OpenGovAus/legistream-backend - @KipCrossing & @king-millez 
+Modular-STVCalculi - https://github.com/MiladKetabGhale/Modular-STVCalculi - @mukeshtiwari
+formalized-voting - https://github.com/mukeshtiwari/formalized-voting/tree/master/SchulzeOCaml  - @mukeshtiwari
+EncryptionSchulze - https://github.com/mukeshtiwari/EncryptionSchulze/tree/master/code/Workingcode - @mukeshtiwari
+OpenAustralia projects - https://github.com/openaustralia - @cofiem
+Australian Twitter News Index https://datastudio.google.com/reporting/1vr1MWHDI91ja9nnxhd3ti3NI0lQWAJDK/page/Q4RO - @cofiem
+CiviWiki - https://github.com/CiviWiki/OpenCiviWiki - @cofiem
+NZ Rules as Code - https://medium.com/@tom_5050/a-rules-as-code-case-study-from-new-zealand-92c7bbc74ba7 - @cofiem
+Aleph Data - https://github.com/alephdata/aleph - @cofiem
+Ushahidi - https://www.ushahidi.com/about - @datakid 
+TDIF - https://www.dta.gov.au/our-projects/digital-identity/trusted-digital-identity-framework - @vteague
+MyGovID - https://www.mygovid.gov.au/ - @vteague
+DigitalID - https://www.digitalid.com/ - @vteague
+Consumer Data Standards - https://consumerdatastandardsaustralia.github.io/standards/ - @reasonableperson
+Electronic Identity eID - https://www.ria.ee/en/state-information-system/electronic-identity-eid.html - @mongrel43
```

Comment:
Republic of Estonia - Information System Authority

Reply:
Sure! Why not? Estonia has shown great leadership in digigov space - Thanks @mongrel43 - great contribution.